src/lib/viewers/doc/_docBase.scss

```diff
@@ -84,7 +84,6 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
         background-color: $d-eight;
         border-radius: $thumbnail-border-radius;
         box-shadow: 0 0 0 1px $off-white, 0 1px 4px 0 rgba(0, 0, 0, .08);
-        transition: box-shadow 50ms ease;
```

Comment:
I noticed some jank when mousing quickly over different thumbnails, so I removed this brief transition. It's perf-related, but I can move it to a separate PR if desired.